<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsSourceSelectBase</class>
 <widget class="QDialog" name="QgsSourceSelectBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>439</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="GroupBox1">
     <property name="title">
      <string>Server connections</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="cmbConnections"/>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnConnect">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>C&amp;onnect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNew">
          <property name="text">
           <string>&amp;New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelete">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>171</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoad">
          <property name="toolTip">
           <string>Load connections from file</string>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="toolTip">
           <string>Save connections to file</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutFilter">
     <item>
      <widget class="QLabel" name="labelFilter">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Fi&amp;lter:</string>
       </property>
       <property name="buddy">
        <cstring>lineFilter</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineFilter">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Display WFS FeatureTypes containing this word in the title, name or abstract</string>
       </property>
       <property name="whatsThis">
        <string>Display WFS FeatureTypes containing this word in the title, name or abstract</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbxUseTitleLayerName">
     <property name="text">
      <string>Use title for layer name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbxFeatureCurrentViewExtent">
     <property name="text">
      <string>Only request features overlapping the current view extent</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbImageEncoding">
     <property name="title">
      <string>Image encoding</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbCRS">
     <property name="title">
      <string>Coordinate reference system</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="labelCoordRefSys">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>441</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnChangeSpatialRefSys">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Change...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbConnections</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnLoad</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>lineFilter</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>cbxUseTitleLayerName</tabstop>
  <tabstop>btnChangeSpatialRefSys</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
